aspnet/security/introduction-to-aspnet-identity.rst

```diff
@@ -0,0 +1,100 @@
+
+	.. image:: introduction-to-aspnet-identity/_static/01-mvc.png
+	
+2. The created project contains the following ASP.NET Identity package.
+
+- ``Microsoft.AspNet.Identity.EntityFramework``
+ The `Microsoft.AspNet.Identity.EntityFramework <http://www.nuget.org/packages/Microsoft.AspNet.Identity.EntityFramework/>`_ package has the Entity Framework implementation of ASP.NET Identity which will persist the ASP.NET Identity data and schema to SQL Server. 
+
+.. note:: In Visual Studio, you can view NuGet packages details by selecting **Tools** -> **NuGet Package Manager** -> **Manage NuGet Packages for Solution**.
+ 
+3. Creating a user.
+
+ Launch the application from Visual Studio (**Debug** -> **Start Debugging**) and then click on the **Register** link in the browser to create a user. The following image shows the *Register* page which collects the user name and password.
+
+	.. image:: introduction-to-aspnet-identity/_static/02-reg.png
+
+When the user clicks the **Register** button, the ``Register`` action of the ``AccountController`` creates the user by calling ``CreateAsync`` function of the ``UserManager`` object, as shown below:
+
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 101-127
+	:emphasize-lines: 4,10
+	:dedent: 8
+	
+4. Log in.
+ If the user was successfully created, they are logged in by the ``SignInAsync`` method, also contained in the ``Register`` action.
+	
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 101-127
+	:emphasize-lines: 19
+	:dedent: 8
+
+The above ``SignInAsync`` method calls the below ``SignInAsync`` task, which is contained in the ``SignInManager`` class. The ``SignInAsync`` method is overloaded. The ``authenticationProperties`` parameter that can be added to the call. This parameter is a dictionary used to store state values about the authentication session. These values are applied to the login and authentication cookie. 
+
+5. Log off.
+ Clicking the **Log off** link calls the ``LogOff`` action in the account controller. 
+ 
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 131-138
+	:emphasize-lines: 5
+	:dedent: 8 
+ 
+The code above shows the ``SignInManager.SignOut`` method. This is analogous to `FormsAuthentication.SignOut <http://msdn.microsoft.com/en-us/library/system.web.security.formsauthentication.signout.aspx>`_ method used by the `FormsAuthentication <http://msdn.microsoft.com/en-us/library/system.web.security.formsauthenticationmodule.aspx>`_ module in Web Forms.
```

Comment:
Consider removing the reference to `FormsAuthentication.SignOut` and spelling out what SignOut actually does. (Clear the cookie?) - I may not be a Web Forms developer.